src/sentry/features/manager.py

```diff
@@ -210,26 +212,49 @@ def has(self, name: str, *args: Any, skip_entity: bool | None = False, **kwargs:
         >>> FeatureManager.has('organizations:feature', organization, actor=request.user)
 
         """
+        sample_rate = 0.01
         try:
-            actor = kwargs.pop("actor", None)
-            feature = self.get(name, *args, **kwargs)
+            with metrics.timer("features.has", tags={"name": name}, sample_rate=sample_rate):
+                actor = kwargs.pop("actor", None)
+                feature = self.get(name, *args, **kwargs)
 
-            # Check registered feature handlers
-            rv = self._get_handler(feature, actor)
-            if rv is not None:
-                return rv
+                # Check registered feature handlers
+                rv = self._get_handler(feature, actor)
+                if rv is not None:
+                    metrics.incr(
+                        "feature.has.result",
+                        tags={"name": name, "result": rv},
+                        sample_rate=sample_rate,
+                    )
+                    return rv
```

Comment:
Is there a function decorator for metrics that will also record the return value of the function? This will avoid having to call `metrics.incr` for every return from this method.

Reply:
I don't think we have a function decorator for that, but it would be helpful to have.